extensions/amp-accordion/1.0/accordion.js

```diff
@@ -136,20 +151,28 @@ export function AccordionSection({
   headerAs: HeaderComp = 'header',
   contentAs: ContentComp = 'div',
   expanded: defaultExpanded = false,
+  animate: defaultAnimate = false,
   header,
   children,
   ...rest
 }) {
   const [id] = useState(generateSectionId);
   const [expandedState, setExpandedState] = useState(defaultExpanded);
+  const contentRef = useRef(null);
+  const hasMountedRef = useRef(false);
 
-  const {registerSection, isExpanded, toggleExpanded} = useContext(
+  const {registerSection, isAnimated, isExpanded, toggleExpanded} = useContext(
```

Comment:
nit: should `isAnimated` and `isExpanded` be named `animated` and `expanded` instead?

Reply:
Forgot to rename `isAnimated`. Just did so. But I kept the `isExpanded` b/c it's actually a function.